packages/dapp/src/pages/Settings.tsx

```diff
@@ -0,0 +1,68 @@
+import { Box, Button, Text } from 'grommet';
+import {useAppState} from "../store";
+
+function maskCharacters(str: string, mask: string, n: number = 1) {
+  // Slice the string and replace with mask, then add remaining string
+  return ('' + str).slice(0, -n).replace(/./g, mask) + ('' + str).slice(-n);
+}
+
+export const Settings = () => {
+  const { account } = useAppState();
+  const { encryptionAccount, switchEncryptionAccount } = useAppState();
+  const { encryptionKey, switchEncryptionKey } = useAppState();
```

Comment:
the same as in the https://github.com/windingtree/org.id-profile-manager/pull/3/files#r796513902